addons/mail/models/mail_channel.py

```diff
@@ -726,13 +726,18 @@ def channel_pin(self, uuid, pinned=False):
         channel = self.search([('uuid', '=', uuid)])
         channel._execute_channel_pin(pinned)
 
+    def channel_pin_for_all_members(self):
```

Comment:
Method should be private by default, unless there is a specific reason to have it public.